docs/source/apis/aws_marketplace.rst

```diff
@@ -0,0 +1,245 @@
+
+AWS Marketplace Deployment Guide
+================================
+
+This guide provides step-by-step instructions for deploying Unstructured API from AWS Marketplace.
+
+Pre-Requirements
+----------------
+
+1. **AWS Account**: Register at `AWS Registration Page <https://aws.amazon.com/>`_, if you don't have an AWS account.
+
+2. **IAM Permissions**: Ensure permissions for ``CloudFormation``.
+
+   - Refer to this `AWS blog post <https://blog.awsfundamentals.com/aws-iam-roles-with-aws-cloudformation#heading-creating-iam-roles-with-aws-cloudformation>`_ to create IAM Roles with CloudFormation.
+
+3. **SSH KeyPair**: Create or use an existing KeyPair for secure access.
+
+   - Follow the ``Create Key Pairs`` in the Amazon EC2 `User Guide <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html>`_.
+
+
+Part 1: Setting-Up a Virtual Private Cloud (VPC)
+------------------------------------------------
+
+1. **Access VPC Dashboard**:
+
+   - In the AWS Management Console, navigate to the VPC service.
+   - Click on “Your VPCs” in the left navigation pane, then “Create VPC.”
+
+2. **Create VPC**:
+
+   - Select ``VPC only``
+   - Enter a ``Name tag`` for your VPC.
+   - Specify the IPv4 CIDR block (e.g., 10.0.0.0/16).
+
```

Comment:
nit: do we need empty line here? it doesn't render on the page afaict?

Reply:
We need the extra line for the rst sub-bullet.